sdk/cosmos/azure-cosmos/test/test_location_cache.py

```diff
@@ -12,19 +12,14 @@
 from azure.cosmos.http_constants import StatusCodes, SubStatusCodes, HttpHeaders
 from azure.cosmos import _retry_utility
 import test_config
-import six
 
 pytestmark = pytest.mark.cosmosEmulator
 
 
 class RefreshThread(threading.Thread):
     def __init__(self, group=None, target=None, name=None,
                  args=(), kwargs=None, verbose=None):
-        if six.PY2:
-            super(RefreshThread, self).__init__(group=group, target=target,
-                                      name=name, verbose=verbose)
-        else:
-            super().__init__()
+        super().__init__()
```

Comment:
hmm I wonder why the previous py2 code was passing in keyword params and the py3 code doesn't?

Reply:
not too sure to be honest, doesn't seem like they were even being set anywhere within the test